django/cantusdb_project/main_app/views/user.py

```diff
@@ -74,6 +76,28 @@ class UserListView(SearchableListMixin, ListView):
     template_name = "user_list.html"
     context_object_name = "users"
 
+class IndexerListView(UserListView):
+    """A list of User objects shown to the public
+
+    This view replaces the indexer list view on the old Cantus. 
+    The indexers are considered a subset of all User objects, the subset shown to the public.
+    This includes the User objects corresponding to Indexer objects on the old Cantus.   
+    When passed a `?q=<query>` argument in the GET request, it will filter users
+    based on the fields defined in `search_fields` with the `icontains` lookup.
+
+    Accessed by /indexers/
+    """
+    template_name = "indexer_list.html"
+    context_object_name = "indexers"
+
+    def get_queryset(self):
+        all_users = super().get_queryset()
+        indexers = all_users.filter(show_in_list=True)
```

Comment:
It would be nice if we could come up with a more descriptive variable than `show_in_list`. It's succinct, but a bit vague. `published` would bring it into alignment with sources/chants/etc., but users aren't exactly published. `publicly_visible` maybe?

Reply:
I'll explain it here and please feel free to give suggestions. All users by default have this `show_in_list` set to `False`. Those who has an Indexer record on the old Cantus will have this set to `True` so they are displayed in the indexer-list page. So this `show_in_list` field indicates whether the user was listed on old Cantus as an Indexer. Not quite the same as `published` or `unpublished`.

Reply:
Perhaps `is_indexer` or `show_as_indexer` will be a better name for the field?

Reply:
I like `is_indexer`!